packages/tss-lib/rollup.config.mjs

```diff
@@ -1,3 +0,0 @@
-export default {
```

Comment:
This is also possibly still necessary, just with an updated path.

Reply:
see above

Reply:
same here, i think the default is fine